general/OKRs/README.md

```diff
@@ -0,0 +1,156 @@
+# Objectives and Key Results
+
+These are the company's OKR for 1Q2018.
+
+Status: Key Results need to be added
+
+Note: the fewer key results the better, the sample ones are just to illustrate where, **not** how many.
+
+### Vision:
+
+* Being the leader of AI on Code
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+### Company Wide:
+
+* Having successful users of source{d}
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Being a better company to work at
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+### Product:
+
+* Launch of hosted playground
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Build a hosted MVP for open-source projects of source{d} PR* (ability to add applications that return results in PR's on GitHub)
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Launch of deduplication of code
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Decide on the faith of legacy projects
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+\* we're lost on the name still, this objective needs its own document to explain better since it's a new product
+
+### Developer Community:
+
+* Create a network of influencers incl. Developers, Companies, Academia
+  * Create a forum (tbd whether it's slack, mailing list, or others) for influencers.
+  * Identify 20 influencers and invite them to join the community.
+  * Meet with all of the influencers that accept the invitation and welcome them personally.
+* Increase our presence in the developer community in Madrid & SF
+  * Organize 2 Go meetups in Madrid
+  * Co-host 2 Go meetups in San Francisco (Holberton school could host)
+  * Creat a Source Code Analysis meetup in San Francisco and run an inaugural event
+* Host and attend conferences and meetups
+  * Source{d} themed talks at 5 conferences or meetups
+  * Source{d} speakers at 5 large (+200 attendees) conferences
+* Keep track of who you met that was relevant to source{d}/relevant new topics in the MLoSC community
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Publish blogposts at a regular pace
+  * Plan weekly schedule for blogs
+  * Publish 8 blog posts during Q1
+* Start cross posting to Medium
+  * Publish 10 posts to Medium
+  * Once we have enough analytics decide whether it's worth cross posting or simply migrating to medium. 
+* Start publishing videos
+  * Create YouTube channel for source{d}
+  * Create monthly schedule for videos and themes
+  * Write 4 detailed scripts
+  * Publish 2 videos
+* Publish a page where we show all source{d} related videos
+  * Redesign talks page
+  * Include all old videos and the ones we'll publish in the meanwhile
+  * Include 5 talks by non source{d} employees
+* Redefine how we use social media and community Slack
+  * Gather statistics of current usage and views on social media
```

Comment:
We already have a dashboard with this metrics right?

Reply:
We have some of those metrics, the first OKR is to make sure we have all of them:
- number of tweets / impressions / followers on twitter
- number of members / messages on slack
etc